### ISSUE TITLE
Ratchet and clank rift apart broken reflections in water

### ISSUE THREAD

**Issue by IGCIT** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#410]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Game is not cracked, modded and use the latest patch

### Game / Application [Required]

Ratchet and clank rift apart

### Game Platform [Required]

- [X] Steam
- [ ] Microsoft Store
- [ ] GOG Galaxy
- [ ] Ubisoft Connect
- [ ] Battle.net
- [ ] Epic Games
- [ ] Rockstar Games Launcher
- [ ] EA app
- [ ] Other (please specify)

### Processor / Processor Number [Required]

i9-11900kb

### Graphic Card [Required]

A770 LE

### GPU Driver Version [Required]

- [X] 31.0.101.4576
- [ ] 31.0.101.4575
- [ ] 31.0.101.4514
- [ ] 31.0.101.4502
- [ ] 31.0.101.4499
- [ ] 31.0.101.4382
- [ ] 31.0.101.4369
- [ ] 31.0.101.4335
- [ ] 31.0.101.4314
- [ ] 31.0.101.4311
- [ ] 31.0.101.4257
- [ ] 31.0.101.4255
- [ ] 31.0.101.4148
- [ ] 31.0.101.4146
- [ ] 31.0.101.4125
- [ ] 31.0.101.4123
- [ ] 31.0.101.4091
- [ ] 31.0.101.4090
- [ ] 31.0.101.4034

### Rendering API [Required]

- [ ] Vulkan
- [ ] OpenGL
- [X] DirectX12
- [ ] DirectX11
- [ ] DirectX10
- [ ] DirectX9

### Windows Build Number [Required]

- [X] Windows 11 22H2
- [ ] Windows 11 21H2
- [ ] Windows 10 22H2
- [ ] Windows 10 21H2
- [ ] Other (Please specify)

### Intel System Support Utility report

[igcit_ssu.txt](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/12183564/igcit_ssu.txt)

### Description and steps to reproduce [Required]

reproducible in planet sargasso

https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/76787191/78cda60d-1bed-41e0-acbc-7157c9617b2c

### Game graphic quality [Required]

- [ ] Low
- [ ] Medium
- [ ] High
- [X] Ultra

### Game resolution [Required]

1920x1080

### Game VSync [Required]

On

### Game display mode [Required]

- [X] Fullscreen
- [ ] Windowed
- [ ] Borderless Windowed

### Detailed game settings [Required]

very-high settings

no ray-tracying
no dyanmic resolution
no upscaler

**Comment by a-One-Fan** [comment]
Same issue as https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/issues/408

**Comment by IGCIT** [comment]
i'm not using ray-tracying tho, glass materials have no glitch with my settings or at least i did not notice any without RT

**Comment by IGCIT** [comment]
yes, i'm using default very high settings, just disabled rt, dynamic resolution and upscaling
TAA is defualt setting

reflections will only appear when camera is moving and disapper when you stop

**Comment by Arturo-Intel** [comment]
Can you share your save files? 

--r2

**Comment by a-One-Fan** [comment]
I'll share mine: 
[rcriftapart_saves_aonefan.zip](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/12196738/rcriftapart_saves_aonefan.zip)

**Comment by IGCIT** [comment]
appears to be fixed in second update

@a-One-Fan i did not try non-TAA, do you still get flickering?

**Comment by a-One-Fan** [comment]
> appears to be fixed in second update
> 
> @a-One-Fan i did not try non-TAA, do you still get flickering?

It does indeed seem to be fixed. Non-TAA methods now have reflections, and I don't see any flickering with TAA and Dynamic Resolution.

https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/100067309/ff152b76-7638-4a72-8912-0c0597401df6
